.github/workflows/ci.yml

```diff
@@ -1,38 +1,26 @@
 name: CI
 
 on:
+  workflow_dispatch:
   push:
     branches:
       - master
+      - run-ci-on-windows
     tags:
       - v*
   pull_request:
     branches:
       - "*"
-  schedule:
-    # Daily at 06:14.  This is just an arbitrary time to stagger
-    # relative to other daily builds in any given organization.
-    - cron: '14 6 * * *'
```

Comment:
Another adjustment for CI in your repo?  I generally encourage people to use the "real" CI, personally.  More consistency, less noise.